_includes/header.html

```diff
@@ -1,23 +1,78 @@
-<nav class="navbar navbar-expand-md navbar-light bg-light noprint">
+<nav class="navbar navbar-expand-md noprint">
+
   <a class="navbar-brand" href="/">
-    <img src="https://static.opensuse.org/chameleon/dist/images/logo/logo-white.svg" class="mr-2" alt="Logo">
-    <span>News</span>
+    <svg class="icon icon-2x mr-2">
+      <use xlink:href="#opensuse"></use>
+    </svg>
+    <span class="navbar-title">News</span>
   </a>
+
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#site-menu">
-    <span class="navbar-toggler-icon"></span>
+    <svg class="icon">
+      <use xlink:href="#menu-line"></use>
+    </svg>
   </button>
+
   <div class="collapse navbar-collapse" id="site-menu">
-    <div class="nav navbar-nav ml-auto">
-      <div class="nav-item">
-        <a class="nav-link" href="{{ "/feed.xml" | prepend: site.baseurl }}">
-          <span class="fa fa-rss"></span>
-          <span class="d-md-none">RSS Feed</span>
+    <ul class="nav navbar-nav mr-md-3">
+      <!-- Categories Dropdown -->
+      <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="cat-menu-link" role="button" data-toggle="dropdown"
+          aria-haspopup="true" aria-expanded="false">
+          <svg class="icon">
```

Comment:
Can't we use fonts for icons? This won't load without js

Reply:
SVG sprite is a better solution than fonts. The most popular reasons are:
1. It is easier to build a SVG sprite by yourself, but hard to build a font. (since we are introducing new icons that the original icon set doesn't include)
2. Fonts don't always load properly. Some mobile browsers by default do not load remote fonts. Then the icon font will be shown as `?` or tofu-like. But JS is more likely to load, since 99% websites use JS. Even the user use NoScript extension, the icon will just be blank, instead of `?` or toufu.
3. Font hinting configuration can affect the appearance of icons.
4. Fonts require one more HTTP request. But SVG sprite can be embedded in either JS or CSS. Less requests result in faster loading speed.

Reply:
Yeah, I'm not opposed to using svg embedded in css, I don't like the js way of doing this however

Reply:
And here is a CSS Tricks article of the comparison https://css-tricks.com/icon-fonts-vs-svg/

Reply:
Alternatively, you can do what the article suggests and use inline svg, instead of using js to do it for you ;) Jekyll has the concept of includes, go ahead and abuse it as much as you need to

Reply:
Having it in JS brings a few benefits:
1. In CSS, we cannot easily make it follow contextual colors. And to get both light/dark icon, the total CSS size will be doubled. And you still won't be able to get a green or red one...
2. CSS load before showing the first page. If we bundle it in CSS, the loading time of first screen can be increased.

The website of https://fontawesome.com/  use SVG icons in JS, too. [js source](https://fontawesome.com/js/11.fce8aa7eb111509d1945.js)

Reply:
If we use inline SVG, it will means every HTML page will have extra bytes to download. With JS, once it is cached, there is no need to download again. It will bring a little big performance benefits.

But the most important thing for me is that I can always deliver the latest icons to CDN. And site integration can be as easy as link a CSS and a JS. Embedding inline SVG requires more manually copy past efforts.

Reply:
SVG isn't that big, and you aren't gonna use megabytes of icons per page, the benefit is negligible at best, and for how much people usually visit our websites, will not sum up to be more than the js used to actually load those icons in the first place

Reply:
Besides, this far, the cdn of the chameleon theme has been disappointing, I would very much prefer having a stable base with versions in urls instead of something that changes in some way every day. We don't have openQA for the theme, when shit breaks we need to fix it on already live website.

Reply:
For some websites I also need to use singular sass elements, so versioned theme would be highly appreciated for that use case too.